Sources/App/Views/PackageController/PackageShow+Model.swift

```diff
@@ -296,11 +296,22 @@ extension PackageShow.Model {
     }
 
     func authorsListItem() -> Node<HTML.ListContext> {
-        guard let authors = authors else { return .empty }
-        let nodes = authors.map { Node<HTML.BodyContext>.a(.href($0.url), .text($0.label)) }
+        guard let authors else { return .empty }
+        var nodes = authors.authors.map { author -> Node<HTML.BodyContext> in
+            if let authorURL = author.url {
+                return .a(.href(authorURL), .text(author.name))
+            } else {
+                return .text(author.name)
+            }
+        }
+
+        if (authors.numberOfContributors > 0) {
+            nodes.append(.text("\(authors.numberOfContributors) other " + "contributor".pluralized(for: authors.numberOfContributors)))
```

Comment:
Should the number here be numberOfContributors minus the number of author nodes shown on the page?

"Written by Author 1, Author 2 and 3 other contributors" where the numberOfContributors is 5?

May be mistaken on what the values are.

edit:// scrap that I've just seen the earlier code where the minus is done. No problem here 😛